tests/test_recursive.py

```diff
@@ -214,3 +214,10 @@ def test_model_serializer(self):
 
         # deserialization
         self.deserialize(RecursiveModelSerializer, representation)
+
+    def test_super_kwargs(self):
+        """RecursiveField.__init__ introspect the parent constructor to pass
+        kwargs properly. read_only is used used here to verify that the
```

Comment:
Correct, this is now updated.

I initially used read_only but a string argument seemed like a better choice.